tensorflow_probability/python/experimental/mcmc/sequential_monte_carlo_kernel.py

```
@@ -138,6 +167,9 @@ def __init__(self,
                propose_and_update_log_weights_fn,
                resample_fn=weighted_resampling.resample_systematic,
                resample_criterion_fn=ess_below_threshold,
+               rejuvenation_fn=rejuvenation_fn,
```

I think this argument should default to `None`, and we can raise an error if `rejuvenation_criterion_fn` is not `None` but `rejuvenation_fn` is `None`.